src/app/components/Curation/HierarchicalGrid/index.test.tsx

```diff
@@ -63,7 +63,7 @@ describe('Hierarchical Grid Curation', () => {
       <HierarchicalGrid headingLevel={headingLevel} summaries={mediaFixture} />,
     );
 
-    const durationString = 'Duration, 2,03';
+    const durationString = ', Duration 2,03';
```

Comment:
As above, remove leading `,`

Reply:
As above, tests fail, due to the fact that the test is fetching the element by its text (and it must match exactly). This change is required.